### PR TITLE
Move logout to controller

### PR DESCRIPTION
**Pull request by LukasReschke**
Testable code. Yay.

cc @blizzz @rullzer

### CODE REVIEW

File: core/routes.php [hunk 1]
```diff
@@ -43,6 +43,7 @@
 		['name' => 'avatar#getTmpAvatar', 'url' => '/avatar/tmp', 'verb' => 'GET'],
 		['name' => 'avatar#postAvatar', 'url' => '/avatar/', 'verb' => 'POST'],
 		['name' => 'login#showLoginForm', 'url' => '/login', 'verb' => 'GET'],
+		['name' => 'login#logout', 'url' => '/logout', 'verb' => 'GET'],
```
Comment (MorrisJobke) [inline core/routes.php]: 😄

Comment (LukasReschke) [inline core/routes.php]: 😉 😜

**Comment by MorrisJobke** [comment]
@LukasReschke I noticed https://github.com/owncloud/core/issues/24057 while testing this. Maybe @DeepDiver1975 could comment on this.

**Comment by rullzer** [comment]
Testable code!!! :dancers: 
Looks good and logout still works :+1:

**Comment by MorrisJobke** [comment]
> @LukasReschke I noticed #24057 while testing this. Maybe @DeepDiver1975 could comment on this.

This is addressed in a separate PR.

👍 for this PR

**Pull request merged**